ci-operator/step-registry/openshift-qe/perfscale-aws-data-path-sg/openshift-qe-perfscale-aws-data-path-sg-commands.sh

```diff
@@ -32,8 +17,7 @@ else
   exit 1
 fi
 
-CLUSTER_ID=$(cat "${SHARED_DIR}/cluster-id")
-CLUSTER_NAME=$(rosa describe cluster -c "${CLUSTER_ID}" -o json | jq -r '.name')
+CLUSTER_NAME=$(oc get infrastructure cluster -o json | jq -r '.status.apiServerURL' | awk -F.  '{print$2}')
```

Comment:
@mukrishn In the past we have had issues where the cluster was up and running but the apiserver url came out blank. We have had several discussions with ocm qe on it. I don't recall receiving a concrete answer or conclusion as to why it was happening and I think eventually it went away on it's own.

Joe even tried to construct the url by hand and code it so that we bypass the issue.
Just a heads up as you are trying to rely on apiServerURL

Reply:
I'd probably avoid relying on apiServerURL, unless you don't have a choice.

Reply:
Okay, lets see. But this task executes right before the data-path workload, it finishes in 22s. If the apiserver was available for the workload in old executions, I hope it should be available a few sec earlier for this task as well. :crossed_fingers:

Reply:
@mukrishn Related discussion and a debug PR for api url being blank that we have been fighting https://github.com/openshift/release/blob/99b368c3eb614a5f122bd80077d8c3db563af465/ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh#L520 
(jfyi)

Reply:
Shouldn't block us but as we keep learning from the related issue, we may need to revisit how we are retrieving CLUSTER_NAME if needed.